french/maps/conv/evelyn_convs.rpy

```diff
@@ -4,131 +4,131 @@
-    e nulla ""
+    e nulla "Disons juste que j'ai une théorie aussi...{w=0.5} En fait, plusieurs, mais la première n'est pas vraiment à moi."
 
 # game/maps/conv/evelyn_convs.rpy:39
 translate french evelyn_outback_cliffs_8d9c6a2f:
 
     # e nulla "One is about the Grimm.{w=0.3} The other is about our master and apprentice duo."
-    e nulla ""
+    e nulla "Une est à propos des Grimms.{w=0.3} L'autre est à propos de notre duo maître et apprenti."
 
 # game/maps/conv/evelyn_convs.rpy:40
 translate french evelyn_outback_cliffs_71474ee2:
 
     # d surprise "Oh.{w=0.5} Are you talking about that paper you and professor Port...?"
-    d surprise ""
+    d surprise "Oh.{=0.5} Est-ce que tu parle de ce papier toi et le professeur Port... ?"
 
 # game/maps/conv/evelyn_convs.rpy:42
 translate french evelyn_outback_cliffs_d1d0a5f5:
 
     # e nulla "Indeed~{w=0.5} Never thought I'd get the chance for some empirical research on the field so soon."
-    e nulla ""
+    e nulla "Évidemment~{w=0.5} Je n'ai jamais pensé que j'aurai la chance pour quelque recherches empiriques sur le terrain de sitôt."
 
 # game/maps/conv/evelyn_convs.rpy:44
 translate french evelyn_outback_cliffs_4e088a55:
 
     # e nulla "Now, if I could only get them to spill the beans.{w=0.3} This would've been so much easier with my old Semblance..."
-    e nulla ""
+    e nulla "Maintenant, si seulement je pouvais leur faire cracher le morceau.{w=0.3} Ç'aurait été bien plus simple avec mon ancienne Semblance..."
 
 # game/maps/conv/evelyn_convs.rpy:45
 translate french evelyn_outback_cliffs_249a1157:
 
     # d amused "And also a gross violation of privacy, bunbun."
-    d amused ""
+    d amused "Et aussi une grossière violation de la vie privée, lapinou."
 
 # game/maps/conv/evelyn_convs.rpy:49
 translate french evelyn_outback_cliffs_1eda7ea2:
 
     # e nulla "Huff!{w=0.3} So glad we're almost done trekking around this place!{w=0.3} The air is {i}not{/i} good for the fur on my ears!"
-    e nulla ""
+    e nulla "Pfiou !{w=0.3} Ravie d'avoir qu'on ait presque fini le tour de cet endroit !{w=0.3} L'air {i}n'est pas{/i} bonne pour la fourrure de mes oreilles !"
 
 # game/maps/conv/evelyn_convs.rpy:50
 translate french evelyn_outback_cliffs_62fda1d8:
 
     # d surprise "And what about your hair...?"
-    d surprise ""
+    d surprise "Et qu'en est t-il de tes cheveux... ?"
 
 # game/maps/conv/evelyn_convs.rpy:52
 translate french evelyn_outback_cliffs_40d0a027:
 
     # e nulla "Oh, that'll be fine.{w=0.3} I have a Zi-Yan for that..."
-    e nulla ""
+    e nulla "Oh, ça ira.{w=0.3} J'ai une Zi-Yan pour ça..."
 
 # game/maps/conv/evelyn_convs.rpy:54
 translate french evelyn_outback_cliffs_ddd112f1:
 
     # e nulla "I also used to have a shampoo that was custom-made for Fauni fur but...{w=0.5} The makers were from Mistral."
-    e nulla ""
+    e nulla "J'ai aussi utilisé le shampooing qui a été fait spécialement pour les Fauni mais...{w=0.5} Les créateurs vennaient de Mistral."
```

Comment:
used to -> habitude
j'avais aussi l'habitude d'utiliser un shampooing fait spécialement pour les Fauni mais...
vennaient (avec un n en trop) -> étaient c'est leur origine